src/systems/a320_systems_wasm/src/spoilers.rs

```diff
@@ -91,5 +91,67 @@ pub(super) fn spoilers(builder: &mut MsfsAspectBuilder) -> Result<(), Box<dyn Er
         Variable::named("HYD_SPOILERS_RIGHT_DEFLECTION"),
     );
 
+    builder.map(
+        ExecuteOn::PostTick,
+        Variable::aspect("HYD_SPOIL_1_RIGHT_DEFLECTION"),
+        |values| values,
+        Variable::named("HYD_SPOILER_1_RIGHT_DEFLECTION"),
+    );
+    builder.map(
+        ExecuteOn::PostTick,
+        Variable::aspect("HYD_SPOIL_2_RIGHT_DEFLECTION"),
+        |values| values,
+        Variable::named("HYD_SPOILER_2_RIGHT_DEFLECTION"),
+    );
+    builder.map(
+        ExecuteOn::PostTick,
+        Variable::aspect("HYD_SPOIL_3_RIGHT_DEFLECTION"),
+        |values| values,
+        Variable::named("HYD_SPOILER_3_RIGHT_DEFLECTION"),
+    );
+    builder.map(
+        ExecuteOn::PostTick,
+        Variable::aspect("HYD_SPOIL_4_RIGHT_DEFLECTION"),
+        |values| values,
+        Variable::named("HYD_SPOILER_4_RIGHT_DEFLECTION"),
+    );
+    builder.map(
+        ExecuteOn::PostTick,
+        Variable::aspect("HYD_SPOIL_5_RIGHT_DEFLECTION"),
+        |values| values,
+        Variable::named("HYD_SPOILER_5_RIGHT_DEFLECTION"),
+    );
+
+    builder.map(
+        ExecuteOn::PostTick,
+        Variable::aspect("HYD_SPOIL_1_LEFT_DEFLECTION"),
+        |values| values,
+        Variable::named("HYD_SPOILER_1_LEFT_DEFLECTION"),
+    );
+    builder.map(
+        ExecuteOn::PostTick,
+        Variable::aspect("HYD_SPOIL_2_LEFT_DEFLECTION"),
+        |values| values,
+        Variable::named("HYD_SPOILER_2_LEFT_DEFLECTION"),
+    );
+    builder.map(
+        ExecuteOn::PostTick,
+        Variable::aspect("HYD_SPOIL_3_LEFT_DEFLECTION"),
+        |values| values,
+        Variable::named("HYD_SPOILER_3_LEFT_DEFLECTION"),
+    );
+    builder.map(
+        ExecuteOn::PostTick,
+        Variable::aspect("HYD_SPOIL_4_LEFT_DEFLECTION"),
+        |values| values,
+        Variable::named("HYD_SPOILER_4_LEFT_DEFLECTION"),
+    );
+    builder.map(
+        ExecuteOn::PostTick,
+        Variable::aspect("HYD_SPOIL_5_LEFT_DEFLECTION"),
+        |values| values,
+        Variable::named("HYD_SPOILER_5_LEFT_DEFLECTION"),
+    );
+
```

Comment:
As we now use directly the internal system values, seems like we don't need those .map at all now.
That'd would need a quick test to be sure though :)